cmd/calyptia/generate.go

```diff
@@ -1,3 +1,4 @@
 package main
 
+//go:generate go run -mod=mod github.com/rjeczalik/interfaces/cmd/interfacer -for github.com/calyptia/api/client.Client -as main.Client -o ./client_gen.go
```

Comment:
Could we or should we replace moq with interfacer? I don't think we should have two of these kinds of tools.

Reply:
interfacer replaces ifacemaker in this case.
ifacemaker uses actual files, while interfacer uses a package, which in this case is the client package.

For context, before we were maintaining the client interface manually. But looks like @niedbalski did generate the client interface using ifacemaker, just I dunno how. Maybe he copied it from the cloud repo manually... dunno.

Reply:
I think he probably did. That was how it was done before. Someone would generate the files with some magic incantation, commit it to the repo and then... :man_shrugging: 

That's why I purposefully had the generate.go file in calyptlia/cloud install ifacemaker with a specific version. Some versions of ifacemaker work with our repos and some do not. I left ifacemaker out of go.mod/go.sum but I don't remember why... probably to make sure the correct version was always installed by generate.go and not let go modules interfere.

Interfacer seems to be a bit more active. It'd be best to see which project does the best compatibility testing.

Let's leave as is for now and add it to the yak shaving pile?

Reply:
The thing is that ifacemaker works in cloud but didn't work in this case for the cli.